crate/operator/grand_central.py

```diff
@@ -479,6 +480,13 @@ async def update_grand_central_deployment_image(
         )
         api_container.image = image
 
```

Comment:
```suggestion
if deployment.spec.template.spec.init_containers:
```
I was playing around with it too and had an old version of gc deployed (from before the `initContainer` was introduced) and was wondering why it exploded :)